cmd/vclusterctl/cmd/import.go

```diff
@@ -47,7 +47,7 @@ vcluster import my-vcluster --cluster connected-cluster \
 		},
 	}
 
-	importCmd.Flags().StringVar(&cmd.Manager, "manager", "", "The manager to use for managing the virtual cluster, can be either helm or platform.")
+	importCmd.Flags().StringVar(&cmd.Driver, "driver", "", "The driver to use for managing the virtual cluster, can be either helm or platform.")
```

Comment:
Just one more removal of the use of "manage" (feel free to ignore)

```suggestion
	importCmd.Flags().StringVar(&cmd.Driver, "driver", "", "The driver for the virtual cluster, can be either helm or platform.")
```